### DISCUSSION TITLE
Consider disabling or removing "force apps to show launcher icons"

### DISCUSSION THREAD

**Discussion by Massi-X** [LSPosed/LSPosed#805] [Q&A] [answered]
Hello,
As the name suggest I can't understand why does lsposed have an option like that? (to always show the app icon even if hidden). I have some modules that hide the app icon and this can cause serious troubles for the users (and even for developers!).

Where did you get the info that after android 10 we can't hide app icons? (And why especially Xposed modules??) This seems more like a decision on your side rather than an Android limitation.

If there is some real motivation for this, please give me a source for this, I suggest to disable it by default anyway to avoid many troubles

**Comment by yujincheng08** [comment]
https://cs.android.com/android/_/android/platform/frameworks/base/+/cf134ebfb716199aa20e19db2e9fdf1d7eb263d5

**Comment by yujincheng08** [comment, marked as answer]
And this https://source.android.com/setup/start/android-10-release#limitations_to_hiding_app_icons

**Comment by Massi-X** [comment]
Ok thank you for your insight. I misunderstood the behavior of the toggle. Thank you, I will add a warning for this
